sdk/src/main/java/com/hedera/hashgraph/sdk/Client.java

```diff
@@ -1004,7 +1005,15 @@ public synchronized void close() throws TimeoutException {
         try {
             network.close();
             mirrorNetwork.close();
+            executor.shutdown();
```

Comment:
Doesn't fix a similar problem in `close(Duration)`. Should combine the two implementations so that one calls the other with a default timeout.